package.json

```diff
@@ -25,14 +33,9 @@
     "react-scripts": "5.0.0",
```

Comment:
`react-scripts` is a dependency of `create-react-app`. I think if we use Vite, we should remove this.